apps/wallet/src/ui/app/pages/approval-request/SignMessageRequest.tsx

```diff
@@ -47,13 +47,11 @@ export function SignMessageRequest({ request }: SignMessageRequestProps) {
 			scrollable
 		>
```

Comment:
Do we need to add blended here as well?

<img width="359" alt="Screenshot 2023-06-27 at 14 03 32" src="https://github.com/MystenLabs/sui/assets/10210143/f68f1d6b-96d1-4f8a-b4c3-9c22d98211c6">